### PR TITLE
test: limit test_no_hotplug_triggered_by_docker to stable releases

### PR DESCRIPTION
**Pull request by aciba90**
## Proposed Commit Message

```
test: limit test_no_hotplug_triggered_by_docker to stable releases

docker repo does not contain pkgs for devel ubuntu series.
```

## Additional Context

https://jenkins.canonical.com/server-team/view/cloud-init/job/cloud-init-integration-noble-ec2/53/testReport/junit/tests.integration_tests.modules/test_hotplug/test_no_hotplug_triggered_by_docker/

## Test Steps

```
CLOUD_INIT_CLOUD_INIT_SOURCE=ppa:cloud-init-dev/daily CLOUD_INIT_PLATFORM=ec2 CLOUD_INIT_OS_IMAGE=noble tox -e integration-tests -- tests/integration_tests/modules/test_hotplug.py::test_no_hotplug_triggered_by_docker

```

## Checklist

- [x] My code follows the process laid out in [the documentation](https://cloudinit.readthedocs.io/en/latest/development/index.html)
- [x] I have updated or added any [unit tests](https://cloudinit.readthedocs.io/en/latest/development/testing.html) accordingly
- [x] I have updated or added any [documentation](https://cloudinit.readthedocs.io/en/latest/development/contribute_docs.html) accordingly

## Merge type

- [x] Squash merge using "Proposed Commit Message"
- [ ] Rebase and merge unique commits. Requires commit messages per-commit each referencing the pull request number (#<PR_NUM>)

### CODE REVIEW

File: tests/integration_tests/releases.py [hunk 1]
```diff
@@ -98,5 +98,7 @@ def from_os_image(
 MANTIC = Release("ubuntu", "mantic", "23.10")
 NOBLE = Release("ubuntu", "noble", "24.04")
 
+UBUNTU_STABLE = (BIONIC, FOCAL, JAMMY, MANTIC)
```
Comment (TheRealFalcon) [inline tests/integration_tests/releases.py:101]: Nit: There's no reason to include bionic here as it is EOL.

**Review by TheRealFalcon — APPROVED** [review]
LGTM!

File: tests/integration_tests/releases.py (hunk 1 above)
Comment (blackboxsw) [reply tests/integration_tests/releases.py:101]: drop it like it's hawwwt

Comment (blackboxsw) [reply tests/integration_tests/releases.py]: squash merged this comment leaving BIONIC out of UBUNTU_STABLE. Will leave dropping all BIONIC integration tests to a separate PR.

**Pull request merged**